plugins/streaming/impls/jms/src/main/java/org/pentaho/di/trans/step/jms/JmsDelegate.java

```diff
@@ -90,10 +90,16 @@
 
   @Injection ( name = "SSL_CONTEXT_ALGORITHM", group = "SSL_GROUP" ) public String sslContextAlgorithm = "";
 
-  @Injection ( name = "IBM_SSL_CIPHERSUITE", group = "SSL_GROUP" ) public String ibmSslCipherSuite = "";
+  @Injection ( name = "SSL_CIPHERSUITE", group = "SSL_GROUP" ) public String sslCipherSuite = "";
```

Comment:
We can't change this because it will break existing ktr's

Reply:
oh, wait, this didn't exist until last sprint.  so should be ok.  nevermind

Reply:
Correct; I added all those parameters last sprint, so the only things that break are some of our test files.